webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -39,12 +74,16 @@ const TaskActionDropdown: React.FC<Props> = ({ task, onComplete, children }: Pro
   ];
 
   if (isKillable) menuItems.unshift({ key: Action.Kill, label: 'Kill' });
+  if (isConnectable()) menuItems.unshift({ key: Action.Connect, label: 'Connect' });
```

Comment:
i think we can wrap code around menuItems with `useMemo`
and we can use `push` instead of `unshift` for performance and readability

Reply:
done.